### DISCUSSION TITLE
Both rotation (*in radians) and rotation degrees properties should be available in the inspector

### DISCUSSION THREAD

**Discussion by Starkandco** [godotengine/godot-proposals#8493] [Editor]
Having rotation be converted from degrees in the inspector leads to confusion for new users some of whom naturally assume the way they interact with this rotation property in script to be the way they interact with it as seen in the inspector.

That is - through degrees e.g. "object.rotation = 180" and expecting an upside down object.

My proposal is to expose both rotation (the radians version) without any conversion, and then separately also expose the rotation degrees property. This will make clear to newer users that there is both rotation and rotation degrees properties available for objects for when they move to scripting.

**Comment by AThousandShips** [comment]
Having two different properties doing the same thing in the editor would be confusing and unnecessary IMO, both are exposed just not in the inspector

**Comment by Calinou** [comment]
This was already proposed in the past and was rejected: https://github.com/godotengine/godot/pull/50042

**Reply by Starkandco** [reply]
If I understand that right it is customisable units but fair enough

**Reply by AThousandShips** [reply]
If the proposal for allowing to switch between the two then having both would likely be out as well (the former would be a better way to implement this IMO)

**Reply by Starkandco** [reply]
I mean it sounds like it came from an issue of complexity which I imagine there is less of with just exposing a existing var in the inspector but I'm new here, I trust you guys know what you're talking about and I'm lacking visibility on a bigger picture here so sorry if I'm coming across annoying
